run-tests/package-lock.json

```diff
@@ -2142,8 +2171,7 @@
     "node_modules/fs.realpath": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
-      "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
```

Comment:
Why did we make these changes to package file?

Reply:
removed unused dependency https://github.com/browserstack/github-actions/pull/37#discussion_r1303995997